src/main/java/com/julienvey/trello/domain/Label.java

```diff
@@ -40,6 +43,30 @@ public Label update() {
         return this;
     }
 
+    /**
+     * @see Trello#addLabelToCard(String, String)
+     */
+    public Label addToCard(String cardId) {
+        getTrelloService().addLabelToCard(cardId, id);
+        return this;
+    }
+
+    /**
+     * @see Trello#addLabelToCard(String, String)
+     */
+    public Label addToCard(Card card) {
+        Objects.requireNonNull(card);
+
+        getTrelloService().addLabelToCard(card.getId(), id);
+
+        List<Label> labels = card.getLabels();
+        if (!labels.contains(this)) {
+            labels.add(this);
+        }
```

Comment:
Label class does not have equals&hashcode, so instances would be compared directly by  object references.
which means two instances of the same label created in memory would be treated as different ones.
we could use comparison by ID to see if the label is in the list.